### PR TITLE
Fix phantomjs

### PR DESCRIPTION
**Pull request by nicholas-l**
Fixes running phantomjs. Couldn't find usage/need of mode=advanced in the examples as it seems to only check for raw, is this an issue?

### CODE REVIEW

**Comment by nicholas-l** [comment]
Once this is done we can remove Function.bind polyfill :)

File: package.json [hunk 1]
```diff
@@ -44,6 +44,7 @@
     "metalsmith-layouts": "1.4.2",
     "nomnom": "1.8.1",
     "pbf": "1.3.5",
+    "phantomjs": "^2.1.2",
```
Comment (ahocevar) [inline package.json]: This is already in `devDependencies`.

File: Makefile [hunk 2]
```diff
@@ -181,7 +181,7 @@ build/timestamps/check-%-timestamp: $(BUILD_HOSTED)/examples/%.html \
                                     $(BUILD_HOSTED)/build/ol.js \
                                     $(BUILD_HOSTED)/css/ol.css
 	@mkdir -p $(@D)
-	./node_modules/.bin/phantomjs --ssl-protocol=any --ignore-ssl-errors=true bin/check-example.js $(addsuffix ?mode=advanced, $<)
+	./node_modules/.bin/phantomjs --ssl-protocol=any --ignore-ssl-errors=true bin/check-example.js $<
```
Comment (ahocevar) [inline Makefile]: Good catch, thanks!

File: package.json (hunk 1 above)
Comment (nicholas-l) [inline package.json]: thats why it rebased so easily, didn;t look much into it :(

**Comment by ahocevar** [comment]
Also, since you based your branch on openlayers:greenkeeper-phantomjs-2.1.2 instead of master, we do not see if Travis runs through with this.

**Comment by nicholas-l** [comment]
Yes, as it says in the greenkeeper pull request that we can work on that for any fixes? I can make a pull request to master but that will fail because this has a dependency of PhantomJS2

**Comment by ahocevar** [comment]
Ok, so please remove the addition of phantomjs to `dependencies`, and then we merge this and see what Travis says on #4717.

**Comment by nicholas-l** [comment]
Already done :)

**Comment by ahocevar** [comment]
Thanks!

**Pull request merged**